src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -533,17 +548,18 @@ class Runner(val testInfo: TestInfo, val suiteRunner: AbstractRunner) {
     // pass if it checks and didn't crash the compiler
     // or, OK, we'll let you crash the compiler with a FatalError if you supply a check file
     def checked(r: CompileRound) = r.result match {
+      case s: Skip => s
       case crash @ Crash(_, t, _) if !checkFile.canRead || !t.isInstanceOf[FatalError] => crash
-      case dnc @ _ => diffIsOk
+      case _ => diffIsOk
```

Comment:
BTW this was intended to be "documentary". The underscore doesn't say why are we at the default case.

Also dnc is not Democratic National Committee but Does Not Compile.